inc/saiqueue.h

```diff
@@ -258,6 +271,32 @@ typedef enum _sai_queue_stat_t
 } sai_queue_stat_t;
 
 /**
+ * @brief Enum defining Queue deadlock event state.
+ */
+typedef enum _sai_queue_pfc_deadlock_event_type_t
+{
+    /** PFC deadlock detected */
+    SAI_QUEUE_PFC_DL_START = 0x00000001,
```

Comment:
please came up with other names cince suffix _START and _END is destined to mark start and end of ranges, and its actually removed by parser, you can use _STARTED, _ENDED